src/styles.scss

```diff
@@ -21,4 +21,88 @@ button.mat-menu-item {
   .mat-step-icon-content .mat-icon svg {
     height: 100%;
     width: 100%;
-  }
\ No newline at end of file
+  }
```

Comment:
Since these all seem to be specific to the new phone control, could they simply be added to the phone control's CSS (phone.component.scss) to keep the "global" scss as small as possible?

Reply:
yeah got it, added the css into component specific css file(phone.component.scss) and updated the PR